lib/theme-config.spec.js

```diff
@@ -1,40 +1,42 @@
-var Code = require('code'),
-    Fs = require('fs'),
-    Hoek = require('hoek'),
-    Lab = require('lab'),
-    Path = require('path'),
-    Sinon = require('sinon'),
-    ThemeConfig = require('./theme-config'),
-    lab = exports.lab = Lab.script(),
-    themePath = Path.join(process.cwd(), 'test/_mocks/themes/valid'),
-    missingVariationsThemePath = Path.join(process.cwd(), 'test/_mocks/themes/missing-variation'),
-    bareBonesThemePath = Path.join(process.cwd(), 'test/_mocks/themes/bare-bones'),
-    describe = lab.describe,
-    expect = Code.expect,
-    it = lab.it,
-    themeConfig;
-
-describe('ThemeConfig', function() {
-    lab.beforeEach(function(done) {
+'use strict';
+
+const Code = require('code');
+const Os = require('os');
+const Fs = require('fs');
+const Hoek = require('hoek');
+const Lab = require('lab');
+const Path = require('path');
+const Sinon = require('sinon');
+const ThemeConfig = require('./theme-config');
+const lab = exports.lab = Lab.script();
+const themePath = Path.join(process.cwd(), 'test/_mocks/themes/valid');
+const missingVariationsThemePath = Path.join(process.cwd(), 'test/_mocks/themes/missing-variation');
+const bareBonesThemePath = Path.join(process.cwd(), 'test/_mocks/themes/bare-bones');
+const describe = lab.describe;
+const expect = Code.expect;
+const it = lab.it;
+
+describe('ThemeConfig', () => {
+    let themeConfig;
+
+    lab.beforeEach(done => {
         themeConfig = ThemeConfig.getInstance(themePath).setVariationByName('First');
 
         done();
     });
 
-    describe('getInstance()', function() {
-        it ('should always return the same instance', function(done) {
+    describe('getInstance()', () => {
+        it ('should always return the same instance', done => {
             expect(themeConfig).to.equal(ThemeConfig.getInstance());
 
             done();
         });
 
-        it ('should allow overwriting of configPath, schemaPath, and variationName by calling getInstance with params', function(done) {
-            var secondConfigPath = '/fake/config.json';
-            var secondSchemaPath = '/fake/schema.json';
-            var secondVariationName = 'Second';
-            var secondThemeConfig;
-
-            secondThemeConfig = ThemeConfig.getInstance('/fake');
+        it ('should allow overwriting of configPath, schemaPath, and variationName by calling getInstance with params', done => {
+            const isWindows = Os.platform() === 'win32';
+            const secondConfigPath = isWindows ? '\\fake\\config.json' : '/fake/config.json';
```

Comment:
i think we use this [path.posix method]|(https://nodejs.org/api/path.html#path_path_posix) instead.  from [this page](https://nodejs.org/api/path.html#path_windows_vs_posix) i think we could do:

```
const normalize = path.posix.normalize;
expect(normalize(secondThemeConfig.configPath)).to.equal(normalize(secondConfigPath));
```